jetty-ee10/jetty-ee10-annotations/src/test/resources/web31true.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <web-app 
-   xmlns="http://xmlns.jcp.org/xml/ns/javaee" 
+   xmlns="https://jakarta.ee/xml/ns/jakartaee"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-   xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+   xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-app_6_0.xsd"
    metadata-complete="true"
-   version="3.1"> 
+   version="6.0">
```

Comment:
again 6.0 content in a 3.1 file?

Reply:
This is a curious one.

Do we keep the version at 3.1, even tho while on Servlet 6.0 there is absolutely no support for Servlet 3.1 as it's across the "big bang" barrier?
Or do we update this "old" descriptor to at least Servlet 5.0?

Reply:
Ah you raised the same question I did above....   I'm thinking we update "old" to be 5.0.  @janbartel agree?